### DISCUSSION TITLE
how

### DISCUSSION THREAD

**Discussion by imdevarsh** [hello-smile6/libv86-terminal-forwarding#2] [Q&A]
plz teach

**Comment by hello-smile6** [comment]
@webdev03 What do you mean?

**Reply by imdevarsh** [reply]
Tell me how to use

**Reply by hello-smile6** [reply]
> Tell me how to use

Open `server.html` in a browser, copy the key, open `index.html` on an iPad Mini 2 or something, and paste in the key. Then, you can have a full terminal on an OS without enough RAM.

**Reply by imdevarsh** [reply]
iPad _Mini_ 2? I have an iPad 2 😄

**Reply by hello-smile6** [reply]
> iPad _Mini_ 2? I have an iPad 2 😄

I actually wrote this with one in mind, by the way. They can't even run Windows 3.1 without crashing, so I wanted a way to have Arch.
